fix: update README.md to include formating command (#138)
jenkins-infra/stories: README.md

# Jenkins User Stories

Previously known as Jenkins is the way. This project showcases real-world Jenkins user stories.

## Deployment

### Deployment Workflow (to be done by Infra or a maintainer)

* Builds are done on `ci.infra.jenkins.io` (VPN needed)
* Successful builds on main branch deploy to netlify
* Netlify site is fronted by fastly

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

For historical record, the admin portal can be reached via https://stories.jenkins.io/admin/

## Development Setup

### System Requirements

> - Node.js (Recommended: Match version in `.tool-versions`)

Reviewer: ```suggestion
* Node.js (Recommended: Match version in `.tool-versions`)
```

> - npm (Node Package Manager)

Reviewer: ```suggestion
* npm (Node Package Manager)
```

## Getting Started

First, fork the repository and clone it.

```bash
git clone <url-from-github>
```

Change Directory

```bash
cd stories
```

Install Dependencies

```bash
npm install
```

Run the development server:

```bash
npm run develop
```

Open [http://localhost:8000](http://localhost:8000) on your browser to see the result

## Code Quality Tools

### Formatting

Ensure consistent code style:

```bash
npm run format
```

## Contributing Guidelines

1. Create feature branches
2. Write clear commit messages
3. Run formatters before submitting PRs

## Contribution Workflow

1. Fork the repository
2. Create a feature branch
3. Make your changes
4. Run prettier formatting
5. Add your changes to staging area and commit your changes
6. Push your changes to GitHub
7. Submit a pull request

> 

Reviewer: ```suggestion
```

## Deployment

### Deployment Workflow (to be done by Infra or a maintainer)

* Builds are done on `ci.infra.jenkins.io` (VPN needed)
* Successful builds on main branch deploy to netlify
* Netlify site is fronted by fastly

For historical record, the admin portal can be reached via https://stories.jenkins.io/admin/

## Development Setup

### System Requirements

- Node.js (Recommended: Match version in `.tool-versions`)
- npm (Node Package Manager)

## Getting Started

First, fork the repository and clone it.

```bash
git clone <url-from-github>
```

Change Directory

```bash
cd stories
```

Install Dependencies

```bash
npm install
```

Run the development server:

```bash
npm run develop

```

Open [http://localhost:8000](http://localhost:8000) on your browser to see the result

## Code Quality Tools

### Formatting

Ensure consistent code style:

```bash
npm run format
```

## Contributing Guidelines

1. Create feature branches
2. Write clear commit messages
3. Run formatters before submitting PRs

## Contribution Workflow

1. Fork the repository
2. Create a feature branch
3. Make your changes

> 4. Run formatters

Reviewer: ```suggestion
4. Run prettier formatting
5. Add your changes to staging area and commit your changes
```

> 5. Submit a pull request

Reviewer: ```suggestion
6. Submit a pull request
```

### Deployment Workflow (to be done by Infra or a maintainer)

* Builds are done on `ci.infra.jenkins.io` (VPN needed)
* Successful builds on main branch deploy to netlify
* Netlify site is fronted by fastly

For historical record, the admin portal can be reached via https://stories.jenkins.io/admin/

## Development Setup

### System Requirements

- Node.js (Recommended: Match version in `.tool-versions`)
- npm (Node Package Manager)

## Getting Started

First, fork the repository and clone it.

```bash
git clone <url-from-github>
```

Change Directory

```bash
cd stories
```

Install Dependencies

```bash
npm install
```

Run the development server:

```bash
npm run develop

```

Open [http://localhost:8000](http://localhost:8000) on your browser to see the result

## Code Quality Tools

### Formatting

Ensure consistent code style:

```bash
npm run format
```

## Contributing Guidelines

1. Create feature branches
2. Write clear commit messages
3. Run formatters before submitting PRs

## Contribution Workflow

1. Fork the repository
2. Create a feature branch
3. Make your changes
4. Run prettier formatting
5. Add your changes to staging area and commit your changes

> 6. Submit a pull request

Reviewer: ```suggestion
7. Submit a pull request
```

> ## Additional Resources

Reviewer: ```suggestion
```

> - [Jenkins Official Website](https://www.jenkins.io/)

Reviewer: ```suggestion
```

Reviewer: Please make sure there are no extra blank lines except one trailing in the end

Author: > blank lines except one trailing in the end

Will do the changes and will let you know when done

# Jenkins User Stories

> Previously known as Jenkins is the way. This project showcases real-world Jenkins implementations and user experiences.

Reviewer: ```suggestion
Previously known as Jenkins is the way. This project showcases real-world Jenkins user stories.
```

Previously known as Jenkins is the way. This project showcases real-world Jenkins implementations and user experiences.

## Deployment

* Builds are done on ci.infra.jenkins.io (VPN Needed)
* Successful builds on main branch deploy to netlify

> * Netlify site is fronted by fastly

Author: hmm, I did try my best.

> * Netlify site is fronted by fastly

Reviewer: Do not always think that changes necessarily improve things

> * Netlify site is fronted by fastly

Author: true

* Netlify site is fronted by fastly

> - Automatic deployment to Netlify for main branch builds

Reviewer: ```suggestion
```

> - Fastly provides frontend caching and performance optimization

Reviewer: ```suggestion
```

> ### Deployment Workflow

Reviewer: ```suggestion
```

- Automatic deployment to Netlify for main branch builds
- Fastly provides frontend caching and performance optimization

> ## Development Setup

Reviewer: ```suggestion
For historical record, the admin portal can be reached via https://stories.jenkins.io/admin/

## Development Setup
```

- Node.js (Recommended: Match version in `.tool-versions`)
- npm (Node Package Manager)

## Getting Started

First, fork the repository and clone it.

```bash
git clone <url-from-github>
```

Change Directory

```bash
cd stories
```

Install Dependencies

```bash
npm install
```

Run the development server:

```bash
npm run develop

> 

Reviewer: ```suggestion
```
